### PR TITLE
crimson/qa: add rados_api_tests.yaml for seastore

### PR DESCRIPTION
**Pull request by liu-chunmei**
Signed-off-by: chunmei-liu <chunmei.liu@intel.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: qa/suites/crimson-rados/seastore/basic/tasks/rados_api_tests.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      osd:
+        osd class load list: "*"
+        osd class default list: "*"
+        osd blocked scrub grace period: 3600
+tasks:
+- workunit:
+    clients:
+      client.0:
+        - rados/test.sh
+          - rados/test_pool_quota.sh
```
Comment (Matan-B) [inline qa/suites/crimson-rados/seastore/basic/tasks/rados_api_tests.yaml:28]: Wrong indentation.
Did you try running against it?

Comment (liu-chunmei) [reply qa/suites/crimson-rados/seastore/basic/tasks/rados_api_tests.yaml]: yes, every test.sh test case passed teuthology test,  But I can't run them at the same time, I meet pg peering issue, it is not the api test itself bug, it is crimson bugs.

**Comment by liu-chunmei** [comment]
jenkins test make check arm64

**Comment by liu-chunmei** [comment]
jenkins test make check arm64

**Comment by liu-chunmei** [comment]
@athanatos could you please take a look on this PR and merge it?

**Review by athanatos — APPROVED** [review]

**Pull request merged**